web/src/components/planner/PlannerFactorySatisfaction.vue

```diff
@@ -120,15 +120,22 @@
                 </span>
               </v-chip>
             </div>
-            <v-chip
-              class="sf-chip yellow"
-              variant="tonal"
-            >
-              <i class="fas fa-bolt" />
-              <span class="ml-2">
-                {{ formatNumber(factory.totalPower) }} MW
-              </span>
-            </v-chip>
+            <v-divider color="#ccc" thickness="2px" />
```

Comment:
```suggestion
            <v-divider class="my-2" color="#ccc" thickness="2px" />
```
Adds some seperation